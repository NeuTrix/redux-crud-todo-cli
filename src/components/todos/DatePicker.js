import React from 'react';
// import PropTypes from 'prop-types';
import TextField from '@material-ui/core/TextField';

const DatePicker = (props) => {

	return (
		<TextField
			{...props}
			fullWidth
			label="set due date"
			margin="dense"
			name="date"
			type="date"
			variant="outlined"
		/>
	);
};

// DatePicker.propTypes = propTypes;

export default DatePicker;
