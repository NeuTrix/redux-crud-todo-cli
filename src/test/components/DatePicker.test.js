/* eslint-env node, mocha */
import React from 'react';
import ReactDOM from 'react-dom';
import { render } from 'enzyme';
import DatePicker from '../../components/todos/DatePicker';

it('renders without crashing', () => {
	const div = document.createElement('div');
	ReactDOM.render(
		<DatePicker
			rank="med"
			// handleChange={f => f}		
		/>, div,
	);
});

const wrapper = render(
	<DatePicker
		rank="med"
		// handleChange={f => f}
	/>,
);

it('...renders the DatePicker ', () => {
	expect(wrapper).toMatchSnapshot();
});
