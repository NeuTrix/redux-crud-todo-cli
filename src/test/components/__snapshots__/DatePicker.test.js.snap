// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`...renders the DatePicker  1`] = `
<div
  class="MuiFormControl-root-1 MuiFormControl-marginDense-3 MuiFormControl-fullWidth-4"
  rank="med"
>
  <label
    class="MuiFormLabel-root-16 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13 MuiInputLabel-marginDense-11 MuiInputLabel-outlined-15"
    data-shrink="false"
  >
    set due date
  </label>
  <div
    class="MuiInputBase-root-36 MuiOutlinedInput-root-23 MuiInputBase-fullWidth-45 MuiInputBase-formControl-37 MuiInputBase-marginDense-43"
  >
    <fieldset
      aria-hidden="true"
      class="MuiPrivateNotchedOutline-root-53 MuiOutlinedInput-notchedOutline-30"
      style="padding-left:8px"
    >
      <legend
        class="MuiPrivateNotchedOutline-legend-54"
        style="width:0.01px"
      >
        <span>
          â€‹
        </span>
      </legend>
    </fieldset>
    <input
      aria-invalid="false"
      class="MuiInputBase-input-46 MuiOutlinedInput-input-31 MuiInputBase-inputType-49 MuiInputBase-inputMarginDense-47 MuiOutlinedInput-inputMarginDense-32"
      name="date"
      type="date"
    />
  </div>
</div>
`;
